test/End_to_end_test/oidc_v1_test.js

```diff
@@ -186,6 +201,16 @@ var apply_test_parameters = (done) => {
   // 2. common endpoint with wrong client secret
   hybrid_config_common_endpoint_wrong_secret = JSON.parse(JSON.stringify(config_template_common_endpoint));
   hybrid_config_common_endpoint_wrong_secret.clientSecret = 'wrong_secret';  
+  // 3. Hybird flow using client assertion with invalid privatePEMKey
+  hybrid_config_clientAssertion_invalid_pemKey = JSON.parse(JSON.stringify(hybrid_config));
+  hybrid_config_clientAssertion_invalid_pemKey.thumbprint = test_parameters.thumbprint;
+  hybrid_config_clientAssertion_invalid_pemKey.privatePEMKey = 'invalid private pem key';
+  hybrid_config_clientAssertion_invalid_pemKey.clientSecret = null;
+  // 4. hybrid flow using client assertion with wrong thumbprint
+  hybrid_config_clientAssertion_wrong_thumbprint = JSON.parse(JSON.stringify(hybrid_config));
+  hybrid_config_clientAssertion_wrong_thumbprint.thumbprint = 'wrongThumbprint';
+  hybrid_config_clientAssertion_wrong_thumbprint.privatePEMKey = test_parameters.privatePEMKey;
```

Comment:
is there a test for an invalid 'privatePEMKey' where the key is valid, but not registered with AzureAD?